TLM/TLM/Manager/Impl/OptionsManager.cs

```diff
@@ -140,10 +156,9 @@ private static void ToSlider(byte[] data, uint idx, ILegacySerializableOption op
                 ToCheckbox(data, idx: 13, PoliciesTab_AtJunctionsGroup.AllowUTurns, false);
                 ToCheckbox(data, idx: 14, PoliciesTab_AtJunctionsGroup.AllowLaneChangesWhileGoingStraight, false);
 
+                ToCheckbox(data, idx: 15, GameplayTab_VehicleBehaviourGroup.DisableDespawning, false);
                 if (dataVersion < 1) {
-                    GameplayTab_VehicleBehaviourGroup.DisableDespawning.Value = !LoadBool(data, idx: 15, true); // inverted
-                } else {
-                    ToCheckbox(data, idx: 15, GameplayTab_VehicleBehaviourGroup.DisableDespawning, false); // not inverted
+                    GameplayTab_VehicleBehaviourGroup.DisableDespawning.Value = !GameplayTab_VehicleBehaviourGroup.DisableDespawning.Value;
                 }
```

Comment:
This alteration is going to trigger change handler twice (when data version is >= 1) - which is why I had the `if .. else` to ensure the checkbox was only getting updated once per deserialisation.